test/client/snapshot-version-request.js

```diff
@@ -144,7 +144,7 @@ describe('SnapshotVersionRequest', function() {
 
     it('errors if asking for a version that does not exist', function(done) {
       backend.connect().fetchSnapshot('books', 'don-quixote', 4, function(error, snapshot) {
-        expect(error.code).to.equal('ERR_OP_VERSION_NEWER_THAN_CURRENT_SNAPSHOT');
+        expect(error).to.be.ok;
```

Comment:
I had to weaken this assertion, but note that if this was running against `sharedb-mongo` (for example), we also wouldn't get the expected error code, since `sharedb-mongo` would [error in `getOps()`](https://github.com/share/sharedb-mongo/blob/0013d15df717ac8af32aa7847ed64bca96be7f66/index.js#L550), which would [trigger an early return](https://github.com/share/sharedb/blob/a5c19b986d65c93abbc9ed11f7a25c0dbaf0788c/lib/backend.js#L830) before hitting the [asserted error](https://github.com/share/sharedb/blob/a5c19b986d65c93abbc9ed11f7a25c0dbaf0788c/lib/backend.js#L836)

Reply:
Okay coverage fixed